lldb/source/Plugins/Process/gdb-remote/ProcessGDBRemote.h

```diff
@@ -301,7 +301,8 @@ class ProcessGDBRemote : public Process,
   using FlashRange = FlashRangeVector::Entry;
   FlashRangeVector m_erased_flash_ranges;
 
-  bool m_vfork_in_progress;
+  // Number of vfork in process.
+  int m_vfork_in_progress;
```

Comment:
Do we really want a number that can go below zero? Doesn't make sense. I would store this as a `uint32_t` and never let it get decremented if it is zero.